scripts/qa_crowbarsetup.sh

```diff
@@ -31,7 +31,7 @@ wanthyperv=
 
 export cloudfqdn=${cloudfqdn:-$cloud.cloud.suse.de}
 export nodenumber=${nodenumber:-2}
-export tempestoptions=${tempestoptions:--t -s}
+export tempestoptions=${tempestoptions:--subunit smoke}
```

Comment:
this means, we will have to adjust the jenkins jobs. What tempestoptions will be needed for a full tempest run?